coveralls.json

```diff
@@ -0,0 +1,17 @@
+{
+  "default_stop_words": [
+    "defmodule",
+    "defrecord",
+    "defimpl",
+    "def.+(.+\/\/.+).+do"
+  ],
+  "terminal_options": {
+    "file_column_width": 40
+  },
+  "coverage_options": {
+    "treat_no_relevant_lines_as_covered": true,
+    "output_dir": "cover/",
+    "minimum_coverage": 5
```

Comment:
For now let's give a small percentage for it till we have enough coverage.

Reply:
sounds good